src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+const Big = require('big.js');
+
+export const operate = (numberOne, numberTwo, operation) => {
+  numberOne = new Big(numberOne);
+  numberTwo = new Big(numberTwo);
+
+  if(operation === '+'){
+    return numberOne.plus(numberTwo)
+  }
+
+  if(operation === '-'){
+    return numberOne.minus(numberTwo);
+  }
+
+  if(operation === '/'){
+    return numberOne.div(numberTwo);
+  }
+
```

Comment:
You have not handled division by ZERO, so it generates an error
Please handle this 
![Screen Shot 2019-12-27 at 1 19 23 PM](https://user-images.githubusercontent.com/40712459/71516588-1cbe3400-28aa-11ea-9edc-47542e092510.png)

Reply:
You can display 'INFINITY' on screen or some other error management strategy you can think of